### PR TITLE
Add version history for 5.8.0

### PR DESCRIPTION
**Pull request by dgault**
For a full list of included PRs see https://github.com/openmicroscopy/bioformats/milestone/58?closed=1&page=1

### CODE REVIEW

File: docs/sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+5.8.0 (2018 February 20)
+------------------------
+
+File format fixes and improvements:
+
+* cellSens VSI
+   - added support for lossless JPEG compression
+* Imspector OBF
+   - improved the parsing of OBF files with embedded OME-XML metadata (thanks to Bjoern Thiel)
+* Ionpath MIBI
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.rst]: This is an entirely new format, isn't it? I think we should list it separately at the top as a new format as per 5.5.0

Comment (sbesson) [reply docs/sphinx/about/whats-new.rst]: Makes sense. In which case we should also move Vectra QPTIFF there.

File: docs/sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+* Micro-Manager
+   - fixed a bug related to the parsing of the metadata closing block
+* NRRD (Nearly Raw Raster Data)
+   - added support for GZIP pixel stream contained within a .nrrd file
+* Olympus OIR
+   - added support for multi file datasets
+* OME-TIFF
+   - when files are ungrouped the dimensions are corrected by checking the indexes for each
+     associated TiffData
+* PerkinElmer Vectra QPTIFF
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.rst]: I'd list separately as a new format too

File: docs/sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,55 @@
...
+   - added a new reader to support the reading of Ionpath Multiplexed Ion Beam Imaging (MIBI)
+     files (thanks to Rachel Finck)
+* Leica LIF
+   - companion metadata files are now attached if present
+* Micro-Manager
+   - fixed a bug related to the parsing of the metadata closing block
+* NRRD (Nearly Raw Raster Data)
+   - added support for GZIP pixel stream contained within a .nrrd file
+* Olympus OIR
+   - added support for multi file datasets
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.rst]: multi-file

**Comment by rleigh-codelibre** [comment]
Other than the points noted by @hflynn, no further changes needed that I can see.

**Comment by hflynn** [comment]
👍

**Comment by hflynn** [comment]
https://ci.openmicroscopy.org/view/Docs/job/BIOFORMATS-DEV-merge-docs/1057/ is green and the built pages render as expected. Good to merge.

**Pull request merged**